frontend/components/InteractiveImage/ImageElements/FlatSolarPanels.tsx

```diff
@@ -0,0 +1,57 @@
+interface Props {
+  top?: string;
+  left?: string;
+  width?: string;
```

Comment:
I'm on the fence about this API. I think it's okay to use Tailwind classes, but it seems like it would be nicer if the caller could just pass in the width or position without having to provide a full class name. For example `width="10%"` instead of `width="w-[10%]"`.

Then instead of setting the `className` we'd set the `style`:

```tsx
<div
  className={`flatSolarpanels ${top ? top : ""} ${left ? left : ""} ${width ? width : ""}`}
  style={{ top, left, width }}
>
  {/* ... */}
</div>
```

What do you think?

```tsx
// Before
<FlatSolarPanels width="w-[10%]" left="left-[16%]" top="top-[12%]" />

// After
<FlatSolarPanels width="10%" left="16%" top="12%" />
```

The same applies in the Windmill component.